src/components/transport_manager/test/transport_manager_impl_test.cc

```diff
@@ -640,7 +642,8 @@ TEST_F(TransportManagerImplTest, UpdateDeviceList_AddNewDevice) {
 
 struct DeviceInfoComparator {
   const transport_manager::DeviceInfo& val_;
-  DeviceInfoComparator(const transport_manager::DeviceInfo& val) : val_(val) {}
+  explicit DeviceInfoComparator(const transport_manager::DeviceInfo& val)
+      : val_(val) {}
 
   bool operator()(const transport_manager::DeviceInfo& val) const {
     return val_.name() == val.name() &&
```

Comment:
@AByzhynar The `DeviceInfoComparator` is used in the two following test to create a matcher using `Truly` to the field `dev_info_`. So if the `operator ==` is predefined then a whole matcher should be written. This will lead to additional lines of code. So I think it is better to use current implementation.